ebu_tt_live/xsd1.1/ebutt_metadata.xsd

```diff
@@ -0,0 +1,621 @@
+				received the instantaneous media for which subtitles were authored and the moment that the authoring tool emitted those subtitles.
+			</xs:documentation>
+		</xs:annotation>
+	</xs:attribute>
+	
+	<xs:attribute name="authorsGroupControlRequest" type="xs:string">
+		<xs:annotation>
+			<xs:documentation>
+				Allows an author to signal their intent to continue providing a stream, or to request that another author begins creating
+				output with a higher ebuttp:authorsGroupControlToken value.
+			</xs:documentation>
+		</xs:annotation>
+	</xs:attribute>
+	
+	<xs:complexType name="sourceMediaIdentifier">
+		<xs:simpleContent>
+			<xs:extension base="xs:string">
+				<xs:attribute name="type" type="xs:string"/>
+			</xs:extension>
+		</xs:simpleContent>
+	</xs:complexType>
+	
+	<xs:complexType name="documentTransitionStyle">
+		<xs:simpleContent>
+			<xs:extension base="xs:string">
+				<xs:attribute name="inUnit" type="ebuttdt:transitionStyleAttributeType" use="required"/>
+				<xs:attribute name="outUnit" type="ebuttdt:transitionStyleAttributeType" use="required"/>
+			</xs:extension>
+		</xs:simpleContent>
+	</xs:complexType>
+	
+	<xs:complexType name="anyMetadata">
+		<xs:annotation>
+			<xs:documentation>Container for metadata information.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:any namespace="##other" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>
+		</xs:sequence>
+	</xs:complexType>
+	
+	<xs:complexType name="metadataBase">
+		<xs:sequence>
+			<xs:element name="title" type="xs:string" minOccurs="0"/>
+			<xs:element name="desc" type="xs:string" minOccurs="0"/>
+		</xs:sequence>
+	</xs:complexType>
+	
+	<xs:complexType name="headMetadata">
+		<xs:complexContent>
+			<xs:extension base="ebuttm:metadataBase">
+				<xs:sequence>
+					<xs:element name="documentMetadata" type="ebuttm:documentMetadata">
+						<xs:unique name="unique-documentFacet">
+							<xs:selector xpath="ebuttm:documentFacet"></xs:selector>
+							<xs:field xpath="."></xs:field>
+						</xs:unique>
+					</xs:element>
+					<xs:element name="agent" type="xs:string" minOccurs="0" maxOccurs="unbounded"/>
+					<xs:element name="binaryData" type="binaryData" minOccurs="0" maxOccurs="unbounded"/>
+				</xs:sequence>
+			</xs:extension>
+		</xs:complexContent>
+	</xs:complexType>
+	
+	<xs:complexType name="divMetadata">
+		<xs:complexContent>
+			<xs:extension base="ebuttm:metadataBase">
+				<xs:sequence>
+					<xs:element name="authoringTechnique" type="authoringTechnique" minOccurs="0" maxOccurs="unbounded"/>
+					<xs:element name="binaryData" type="binaryData" minOccurs="0" maxOccurs="unbounded"/>
+					<xs:element name="transitionStyle" type="xs:string" minOccurs="0" maxOccurs="unbounded"/>
+					<xs:element name="facet" type="facet" minOccurs="0" maxOccurs="unbounded"/>
+				</xs:sequence>
+			</xs:extension>
+		</xs:complexContent>
+	</xs:complexType>
+	
+	<xs:complexType name="pMetadata">
```

Comment:
After further discussion, facet is permitted by anyMetadata and is defined, so it's a no-op to specifically add it in.